tests/test_markitdown.py

```diff
@@ -341,8 +361,8 @@ def test_markitdown_llm() -> None:
 
 if __name__ == "__main__":
     """Runs this file's tests from the command line."""
-    test_markitdown_remote()
-    test_markitdown_local()
+    # test_markitdown_remote()
+    # test_markitdown_local()
     test_markitdown_exiftool()
-    test_markitdown_deprecation()
-    test_markitdown_llm()
```

Comment:
Hi, I noticed that the tests in test_markitdown_exiftool() were commented out in this commit. Was this an intentional change, or was it possibly included by mistake?